doc/03-flows.md

```diff
@@ -0,0 +1,41 @@
+# Flows
+
+Status: (alpha)
```

Comment:
What does that mean?

Reply:
OK so this means "API might change, there be dragons"

Some places call "untested experimental code" alpha, that's why I asked